tide-whisperer.go

```diff
@@ -99,6 +117,23 @@ func main() {
 		return !(perms["root"] == nil && perms["view"] == nil)
 	}
 
+	//log error detail and write as application/json
+	jsonError := func(res http.ResponseWriter, err detailedError, startedAt time.Time) {
+
+		if err.RawMessage == "" {
```

Comment:
so - don't bother with the copy or even having the RawMessage filed and just pass the original error if one exists and log that 

e.g. 
```
jsonError := func(res http.ResponseWriter, err detailedError, internal error, startedAt time.Time) {
    ...

    log.Println(DATA_API_PREFIX, fmt.Sprintf("[%s] [%s] failed after [%.5f]secs with error [%s] ", err.Id, err.Code, time.Now().Sub(startedAt).Seconds(), internal.Error()))

    ...

}
```